docs/modules/plugins/pages/plugin-azure-event-grid.adoc

```diff
@@ -74,6 +74,17 @@ include::partial$azure-profile-and-subscription.adoc[leveloffset=+1]
 
 ==== Collect system topics info
 
+[WARNING]
+====
+The step is deprecated and will be removed in VIVIDUS 0.6.0. The replacement is
+xref:plugins:plugin-azure-resource-manager.adoc#_get_information_about_azure_resource[the generic step retrieving information about Azure resources].
+The replacement pattern is:
```

Comment:
```suggestion
The replacement step is:
```
??

Reply:
it's actually "replacement pattern", see the details, the placeholders are used input parameters